documentation/pages/move-registry/tooling/mvr-cli.mdx

```diff
@@ -0,0 +1,45 @@
+import { Tabs } from 'nextra/components'
+import { Callout } from 'nextra/components'
+
+# MVR CLI
+
+MVR CLI is used to manage your package dependencies and to interact with the Move Registry. It is a command-line tool that allows you to:
+
+1. Add dependencies to your Move project.
+2. Build your Move project with MVR dependencies.
+
+
+## Installation
+
+To install the MVR CLI, you can use the following command:
+
+```bash
+TODO: add installation command
+```
```

Comment:
we need to figure out how to do this. The simplest for now is to have it via releases. We can also add a script for downloading and installing it, similar to what Walrus used to have.

Simplest version for us:
Go to https://github.com/mystenLabs/mvr/releases and download the binary corresponding to your architecture and OS. Put the binary on the PATH and then use `mvr --help` to see all commands.